gsad/src/gsad_http.c

```diff
@@ -465,38 +465,78 @@ char *
 gsad_message_new (const char *title, const char *msg,
                   cmd_response_data_t *response_data)
 {
-  gchar *xml, *resp;
-
-  xml = g_strdup_printf ("<gsad_response>"
-                         "<title>%s (GSA %s)</title>"
-                         "<message>%s</message>"
-                         "<token>%s</token>"
-                         "</gsad_response>",
-                         title,
-                         GSAD_VERSION,
-                         msg,
-                         "");
+  gchar *html;
+
+  html = g_strdup_printf (
+    "<!DOCTYPE html PUBLIC \"-//W3C//DTD XHTML 1.0 Transitional//EN\" \"http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd\">"
+    "<html xmlns=\"http://www.w3.org/1999/xhtml\" lang=\"\">"
+    "<head>"
+      "<meta http-equiv=\"Content-Type\" content=\"text/html; charset=UTF-8\"></meta>"
+      "<link rel=\"icon\" href=\"/img/favicon.gif\" type=\"image/gif\"></link>"
+      "<title>Greenbone Security Assistant</title>"
+      "<link rel=\"stylesheet\" type=\"text/css\" href=\"/css/gsa-base.css\"></link>"
```

Comment:
You can remove most of the css links and script tags. Especially the js files below aren't included in gsa(d) anymore.

Reply:
I intentionally copied over the original transformation output for documentation purpose.
Once the other calls of xsl_transform() are resolved, the HTML parts can be simplified
in conjunction with the actual removal of the then-unused css and js files.